Framework/Geometry/src/Rendering/ShapeInfo.cpp

```diff
@@ -34,6 +35,37 @@ void ShapeInfo::getObjectGeometry(ShapeInfo::GeometryShape &shape,
   height = m_height;
 }
 
+ShapeInfo::CuboidGeometry ShapeInfo::cuboidGeometry() const {
+  assert(m_shape == GeometryShape::CUBOID);
+  CuboidGeometry g{m_points[0], m_points[1], m_points[2], m_points[3]};
```

Comment:
I think we can do without the temporary as this is just aggregate initialization so:
```c++
 return {m_points[0], m_points[1], m_points[2], m_points[3]};
```
directly?

Reply:
Good point.